packages/accordion/package.json

```diff
@@ -35,11 +35,11 @@
     "@availity/mui-button": "workspace:^",
     "@availity/mui-layout": "workspace:^",
     "@availity/mui-typography": "workspace:^",
-    "@mui/material": "^5.11.9",
+    "@mui/material": "^5.15.15",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "tsup": "^5.12.7",
-    "typescript": "^4.6.4"
+    "tsup": "^8.0.2",
+    "typescript": "^5.4.5"
```

Comment:
We should also update these in the nx-generator package.json__template__

Everything else lgtm

Reply:
Good call, fixed.